news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -0,0 +1,124 @@
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/elpis.jpg)](https://osu.ppy.sh/s/559260)
+
+After having his [first ever ranked map](https://osu.ppy.sh/s/507007) featured in Spotlights at the start of the year, iloveyou4ever's unique mapping style continues to impress, and his latest map ELPIS certainly showcases his growing creativity and talent as a mapper. This kind of progression is exactly the kind we are looking for, and thus he has earned a rightful spot on this month's Spotlights!
+
+The key defining feature of this map lies in iloveyou4ever's excellent choice of patterns. In particular, his placement of triplets is nothing short of beautiful, as it captures the swell and subtle nuances of the song very well. The streams throughout the map are also fairly varied, carrying good flow in spite of the fluctuating BPM. All in all another excellent addition to the ranked section, and I personally cannot wait to see more from him!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/best-day.jpg)](https://osu.ppy.sh/s/316779)
+
+A solid mapset in this month, from Kantan to Muzukashii,Volta did very good job by following a lot of instrument(including vocal) and assigned drum and beat as main role in order to maintain the map's structure, then used other one like vocal to decorate the map to make the map become more fun to play. Moreover, Nardo's Oni also rather decent, every hitsounds usage almost mapped in the best way. After looking how perfect Oni is, I just wonder how Inner Oni can be mapped in good way by expanding the note's density, but I am totally wrong,it indeed contains much more notes expansion and has lot of continuous 1/2+1/4 patterns,but every 1/2 hitsounds are correct mapped and every 1/4 patterns placed in perfect section.Furthermore, Volta used some harder 1/4 pattern to make the map more fun to play.I'd say that I enjoy this map very much!
+
+## osu!catch
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/chronoxia.jpg)](https://osu.ppy.sh/s/429184)
+
+Our top pick is something that many osu!catch players would be familiar with due to it being selected for the osu!catch World Cup 2016's Finals mappool, and is now a fully ranked map. Both the Overdose made by Furely and the Chronos difficulties feature patterns that we don't usually see in most ranked maps - strict antiflow patterns, high overall spacing, and generally a lot of places to miss - while still making it a fun mapset of fair difficulty. Overall, the map is quite challenging for its star rating, being harder than other mapsets people usually see in the Ranked category. It also features several mappers, providing players with different styles across all difficulties. It's quite the engaging map, and one that could surely make you wiggle from joy!
```

Comment:
Also link to the profiles of all usernames referenced, maybe even reference the CWC 2016 wiki with the mappool header as well.